lxd/patches.go

```diff
@@ -85,6 +85,7 @@ var patches = []patch{
 	{name: "storage_unset_invalid_block_settings", stage: patchPostDaemonStorage, run: patchStorageUnsetInvalidBlockSettings},
 	{name: "candid_rbac_remove_config_keys", stage: patchPreDaemonStorage, run: patchRemoveCandidRBACConfigKeys},
 	{name: "storage_set_volume_uuid", stage: patchPostDaemonStorage, run: patchStorageSetVolumeUUID},
+	{name: "storage_set_volume_uuid_with_query", stage: patchPostDaemonStorage, run: patchStorageSetVolumeUUIDWithQuery},
```

Comment:
can we call this `storage_set_volume_uuid_v2` as `_with_query` is a bit confusing imho (they all use queries).

Reply:
Yes sure.

Reply:
Fixed.